lib/services/role.go

```diff
@@ -1299,16 +1279,15 @@ func (set RoleSet) GetLoginsForTTL(ttl time.Duration) (logins []string, matchedT
 // CheckAccessToRemoteCluster checks if a role has access to remote cluster. Deny rules are
 // checked first then allow rules. Access to a cluster is determined by
 // namespaces, labels, and logins.
-//
-// Note, logging in this function only happens in debug mode, this is because
-// adding logging to this function (which is called on every server returned
-// by GetRemoteClusters) can slow down this function by 50x for large clusters!
 func (set RoleSet) CheckAccessToRemoteCluster(rc types.RemoteCluster) error {
```

Comment:
Any way to roll this into `CheckAccess`?

Reply:
I looked briefly and some of the extra backwards compatibility logic in there made me hesitant to change it. I will take a closer look and follow up with a subsequent PR if I can find a way to remove this.